### PR TITLE
Updating circle ci to use go 1.10.1 during tests

### PR DESCRIPTION
**Pull request by wstewartii**

### CODE REVIEW

File: tools/builders/launcher-builder/1.10.1/cloudbuild.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+steps:
+- name: 'gcr.io/cloud-builders/git'
```
Comment (groob) [inline tools/builders/launcher-builder/1.10.1/cloudbuild.yml]: I don't think we need the clone steps here. It's just a Dockerfile that is being submitted. Let's have the makefile use `gcloud container builds submit --config cloudbuild.yaml .`.

Comment (wstewartii) [reply tools/builders/launcher-builder/1.10.1/cloudbuild.yml]: container can't see the Dockerfile without cloning the repo.

Comment (groob) [reply tools/builders/launcher-builder/1.10.1/cloudbuild.yml]: Why not? it should get uploaded when you submit.

Comment (wstewartii) [reply tools/builders/launcher-builder/1.10.1/cloudbuild.yml]: actually looks like I may be able to pass "." to get it to work.

**Pull request merged**